src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -83,7 +83,7 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig kv import', arg_group='File') as c:
         c.argument('path', help='Local configuration file path. Required for file arguments.')
-        c.argument('format_', options_list=['--format'], arg_type=get_enum_type(['json', 'yaml', 'properties']), help='Imported file format. Required for file arguments. Currently, feature flags are only supported in json format.')
+        c.argument('format_', options_list=['--format'], arg_type=get_enum_type(['json', 'yaml', 'properties']), help='Imported file format. Required for file arguments. Currently, feature flags are not supported in properties format.')
```

Comment:
curious why format_ has a '_' in the end

Reply:
Because 'format' is a built-in identifier (pylint throws an error)

---
In reply to: [362150723](https://github.com/Azure/azure-cli/pull/11637#discussion_r362150723) [](ancestors = 362150723)